### DISCUSSION TITLE
How to get visuals for anymal_c.usd taken from omniverse nucleus cloud?

### DISCUSSION THREAD

**Discussion by ihlej** [isaac-sim/IsaacLab#890] [Q&A]
I got anymal_c.usd from http://omniverse-content-production.s3-us-west-2.amazonaws.com/Assets/Isaac/4.0/Isaac/IsaacLab/Robots/ANYbotics/ANYmal-C/anymal_c.usd
[make_instanceable.txt](https://github.com/user-attachments/files/16783267/make_instanceable.txt)

and I can see all the components in the Stage. However, the visuals are red in Stage and give a warning. The parts of the warning that I can see are below (part of the warning is invisible due to its length): 
`Warning: in _ReportErrors at line 2890 of /buildAgent/work/ac88d7d902b57417/USD/pxr/usd/usd/stage.cpp -- In </anymal/base/visuals>: Could not open asset @/Olympus-ws/Other-USD/Props/instanceable_meshes.usd@ for reference`

I can see that Props/instanceable_meshes.usd doesn't exist, however, I'm uncertain where I can get these from. 

## What I've tried

I have a script to make USDs instanceable that has worked for my own USDs, as I can see my own USDs just fine. I've tried adjusting the Asset Path to match my created `anymal_c_instanceable_meshes.usd`. This removes the warning, however, still no visuals. 

## My question

How am I supposed to see anymal_c.usd? Do I have to make it instanceable? Is there something wrong with my instanceable script? 

## Screenshots, etc.
Out of the box having only downloaded the anymal_c USD:
![image](https://github.com/user-attachments/assets/51c603ad-b9d2-4f05-83bc-d8f1cd6eaaf6)
Trying my own intanceable mesh:
![image](https://github.com/user-attachments/assets/91804c55-404a-4d4a-97ca-c26300cc5cf4)

**Comment by ihlej** [comment]
My make_instanceable script in text form because uploading .py isn't supported. 
[make_instanceable.txt](https://github.com/user-attachments/files/16783286/make_instanceable.txt)

**Comment by Dhoeller19** [comment]
Hi @Aemperior,
I would get the latest Anymal C usd here: http://omniverse-content-production.s3-us-west-2.amazonaws.com/Assets/Isaac/4.2/Isaac/IsaacLab/Robots/ANYbotics/ANYmal-C/anymal_c.usd.
The instanced parts are referenced here: http://omniverse-content-production.s3-us-west-2.amazonaws.com/Assets/Isaac/4.2/Isaac/IsaacLab/Robots/ANYbotics/ANYmal-C/Props/instanceable_meshes.usd.
So you would have to download this USD as well, and put it in a folder `Props` that lives in the same folder as `anymal_c.usd`.
